### PR TITLE
Fix Live TV tiles aspect ratio in CardPresenter.java

### PR DESCRIPTION
**Pull request by MrChip53**
**Changes**
Fix Live TV tile aspect ratio.

**Issues**
#575 

...you be the judge 😅

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java [hunk 1]
```diff
@@ -90,7 +91,10 @@ public void setItem(BaseRowItem m, String imageType, int lHeight, int pHeight, i
                     } else if (imageType.equals(ImageType.THUMB)) {
                         aspect = ImageUtils.ASPECT_RATIO_16_9;
                     } else {
-                        aspect = Utils.getSafeValue(ImageUtils.getImageAspectRatio(itemDto, m.getPreferParentThumb()), ImageUtils.ASPECT_RATIO_7_9);
+                        if (itemDto.getBaseItemType() == BaseItemType.UserView)
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java:94]: Make this an `else if`

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

**Comment by thornbill** [comment]
This stretches out the default card images used in the app. We should probably have different images for these that use the proper aspect ratio.

**Comment by MrChip53** [comment]
Maybe we could find another fix that changes aspect ratio only if a custom image is defined. I'll look into it later.

File: app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java (hunk 1 above)
Comment (MrChip53) [reply app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java:94]: Completely changed the method used due to the issue raised below.

**Comment by MrChip53** [comment]
@thornbill @nielsvanvelzen I believe this fix is more proper and addresses the stretched image issue

**Review by thornbill — APPROVED** [review]
:+1: This seems like a good approach to me.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**